manifests/init.pp

```diff
@@ -164,10 +175,33 @@
     restart    => '/usr/bin/systemctl reload nftables',
   }
 
+  if $allow_unmanaged_rules {
+    file { $inmem_rules_hash_file:
```

Comment:
do we really need to remove it by default?

Reply:
It's mainly to remove garbage if the user switches the setting.